docs/7.Scan Examples/Sca.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: default
+published: true
+title: SCA scanning
+nav_order: 20
+---
+
+# SCA scanning
+Checkov is an SCA (Software Composition Analysis) tool. This means it scans package files closed source images for Common Vulnerabilities and Exposures (CVEs).
```

Comment:
Recommend: `Checkov is an SCA (Software Composition Analysis) tool. This means it scans package files and container images for Common Vulnerabilities and Exposures (CVEs).`